lib/Models/Terria.ts

```diff
@@ -533,6 +533,7 @@ export default class Terria {
   @observable depthTestAgainstTerrainEnabled = false;
 
   @observable stories: StoryData[] = [];
+  @observable storyPromptShown: number = 0;
```

Comment:
Might be good to leave a comment here on the purpose of this property.

Reply:
Definitely needs a comment. This sort of property should be on `ViewState` instead of `Terria`, but currently the origin of the scenes in `terria.stories` cannot be determined after it's been set.

Reply:
Added comment for this line. Cannot move to ViewState without a bigger fix I think

Reply:
Maybe sometime we'll fix story scene provenance (probably when we next extend stories, either with having a catalogue of stories for a map, or logging on to author official stories) and then we can get rid of this and detect loaded stories from outside Terria.ts.